lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialServiceTest.java

```diff
@@ -209,9 +227,8 @@ void shouldCreateValidSignedJWT() throws JOSEException {
         assertThat(
                 jwtClaimsSetCaptor.getValue().toString(),
                 containsString("\"verificationScore\":0"));
-        assertThat(
```

Comment:
I think it's still valid to make an assertion about the presence of the `txn` field ??
You can add your ci assertion rather than replacing this one